### PR TITLE
SPCTR-2309 Confirmation popup for svg upload

### PR DESCRIPTION
**Pull request by adi3890**
### Description
- Adds a confirmation message for the first time to allow SVGs

### Checklist:
- [ ] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

**Review by AbhijitNage123 — APPROVED** [review]

File: lib/gutenberg-templates/classes/class-ast-block-templates.php [hunk 1]
```diff
@@ -68,8 +68,6 @@ public function custom_upload_mimes( $mimes ) {
 
 			// Allow JSON files.
 			$mimes['json'] = 'application/json';
-			// Allows svg files.
-			$mimes['svg'] = 'image/svg+xml';
```
Comment (RajkiranBagal) [inline lib/gutenberg-templates/classes/class-ast-block-templates.php:72]: i think we should not remove this. if user never uses icon block then he may get issues with templates import

**Review by RajkiranBagal — APPROVED** [review]

File: .eslintrc.js [hunk 2]
```diff
@@ -3,6 +3,7 @@ module.exports = {
 	extends: [ 'plugin:@wordpress/eslint-plugin/recommended' ],
 	ignorePatterns: [ 'deprecated.js' ],
 	rules: {
+		'no-alert': 'off',
 		'camelcase': 'off',
```
Comment (RajkiranBagal) [inline .eslintrc.js:7]: Are these changes still needed?

File: classes/class-uagb-init-blocks.php [hunk 3]
```diff
@@ -72,9 +72,22 @@ public function __construct() {
...
 		}
 
+		if ( current_user_can( 'edit_posts' ) ) {
+			add_action( 'wp_ajax_uagb_svg_confirmation', array( $this, 'confirm_svg_upload' ) );
+		}
+
 		// Needs this as we need to upload svg files for svg picker.
 		add_filter( 'upload_mimes', array( $this, 'custom_upload_mimes' ) ); // phpcs:ignore WordPressVIPMinimum.Hooks.RestrictedHooks.upload_mimes
 
+		// temporary code for testing purpose of QA. need to remove before merging.
```
Comment (RajkiranBagal) [inline classes/class-uagb-init-blocks.php:82]: Is this code needed?

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: .eslintrc.js (hunk 2 above)
Comment (adi3890) [reply .eslintrc.js]: Thanx for reminding, removed this.

File: classes/class-uagb-init-blocks.php (hunk 3 above)
Comment (adi3890) [reply classes/class-uagb-init-blocks.php]: removed

File: src/components/popup-confirm/index.js [hunk 4]
```diff
@@ -0,0 +1,96 @@
+import { Button, Modal } from '@wordpress/components';
+import React, { useLayoutEffect, useState } from 'react';
```
Comment (RajkiranBagal) [inline src/components/popup-confirm/index.js:2]: Import from @wordpress/element & remove React import.

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: src/components/popup-confirm/index.js (hunk 4 above)
Comment (adi3890) [reply src/components/popup-confirm/index.js]: done

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**